src/node_file.cc

```diff
@@ -1681,7 +1683,7 @@ static void ReadDir(const FunctionCallbackInfo<Value>& args) {
         return;
       }
 
-      name_v.push_back(filename.ToLocalChecked());
+      name_v.push_back(filename.FromMaybe(Local<Value>()));
```

Comment:
I’d have a preference to keep this one and the preceding ones as they are, because `.ToLocalChecked()` is the right thing to use *if* it has been checked that the contents are not empty.